src/components/Modal/__stories__/modal.stories.mdx

```diff
@@ -0,0 +1,308 @@
+import Modal from "../Modal";
+import { useModalHelper } from "./helpers";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createStoryMetaSettings } from "../../../storybook/functions/create-component-story";
+import "./Modal.stories.scss";
+import { ModalContent, ModalFooter, ModalHeader, TextField } from "components";
+import { Settings } from "components/Icon/Icons";
+
+export const metaSettings = createStoryMetaSettings({
+  component: Modal,
+  enumPropNamesArray: [], // List enum props here
+  iconPropNamesArray: [], // List props that are typed as icons here
+  actionPropsArray: [] // List the component's actions here
+});
+
+<Meta title="Feedback/Modal" component={Modal} argTypes={metaSettings.argTypes} decorators={metaSettings.decorators} />
+
+<!--- Component documentation -->
+
+# Modal
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+
+An easy to use accessible Modal component
+
+<Canvas>
+  <Story name="Overview">
```

Comment:
This specific modal button doesn't reopen modal once it was closed
Steps to reproduce: 
1) Focus button via Tab
2) Press Space to open Modal
3) Press Esc to close Modal
4) Result: modal doesn't opens again

Reply:
fixed, thanks!